.github/workflows/ci.yaml

```diff
@@ -0,0 +1,67 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 2.15
+      - 2.16
+      - 3.0
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        operating-system:
+          - ubuntu-latest
+        php-version:
+          - 7.1
+          - 7.2
+          - 7.3
+          - 7.4
+        dependencies: ['--ignore-platform-req=hhvm']
```

Comment:
GitHub's VM has HHVM, but `phpunitgoodpractices/polyfill` does not want to be installed with HHVM.